app/models/dttp/placement_assignment.rb

```diff
@@ -39,10 +39,6 @@ def route_dttp_id
       response["_dfe_routeid_value"]
     end
 
-    def lead_school_id
```

Comment:
Not sure about this as the deals with `jsonb`

cc @stevehook can this class be deleted altogether given it's dttp related?

Reply:
Yeah I did wonder if we could just delete the whole thing, but I wasn't sure. @stevehook what do you think?

Reply:
Would we not have the historical data in a database backup somewhere? If they're not currently needed by the app, I would suggest we delete them. I could do that in another PR maybe. @tomtrentham what do you think?

Reply:
@adamjcolvin This feels separate to the current work to me. I think it's more closely related to the DTTP removal work that we've got in the tech debt backlog so I reckon we should leave this file now.

Reply:
@tomtrentham no worries, I've undone this change for now then 👍